bin/build_cmd.ml

```diff
@@ -202,9 +202,20 @@ let fmt =
     ]
   in
   let term =
-    let+ common = Common.term in
+    let+ common = Common.term
+    and+ no_promote =
+      Arg.(
+        value & flag
+        & info [ "no-promote" ]
```

Comment:
Maybe the flag should be called `preview`?

Reply:
yep, changed